gulp/config.js

```diff
@@ -21,6 +21,7 @@ module.exports = () => {
     ],
     meta: ['src/meta/**/*'],
     definitions: ['src/**/definition.yml'],
+    icons: ['src/materials/icons/*.svg'],
```

Comment:
Ah it's intended for https://github.com/ginetta/skeleton/blob/081b1e659859bc0343e836aa78ca75952e383388/src/materials/icons/layout.pug#L32

But that's a bit weird to me because the icons are handles in two scripts.
I'd prefer the single icons move to be also done in the icons task. This would also save a duplicate disc read for each icons.